harvester.py

```diff
@@ -50,8 +52,9 @@ def read_xml(terminology):
                     with open(file_abs_path, 'wb') as f:
                         f.write(req_main.content)
                     # write down the corresponding ETag of a collection into .ini file
-                    header_ETag = head.headers['ETag']
-                    add_config_ETag(config_file_name, collection_name, header_ETag)
+                    #header_ETag = head.headers['ETag']
```

Comment:
could you revert this change, as this disables incremental harvesting?

Reply:
when enabled, code fails to run because ETAg / etag seems to have been stripped from the header (KeyError: 'etag')